core/app/[locale]/layout.tsx

```diff
@@ -110,23 +96,23 @@ export default async function RootLayout({ params, children }: Props) {
   const messages = await getMessages();
 
   return (
-    <html
-      className={clsx(inter.variable, dm_serif_text.variable, roboto_mono.variable)}
-      lang={locale}
-    >
-      <head>
-        <DraftModeScript appOrigin={process.env.MAKESWIFT_APP_ORIGIN} />
-      </head>
-      <body>
-        <Notifications />
-        <NextIntlClientProvider locale={locale} messages={messages}>
-          <NuqsAdapter>
-            <Providers>{children}</Providers>
-          </NuqsAdapter>
-        </NextIntlClientProvider>
-        <VercelComponents />
-      </body>
-    </html>
+    <MakeswiftProvider previewMode={(await draftMode()).isEnabled}>
+      <html className={clsx(fonts.map((f) => f.variable))} lang={locale}>
+        <head>
+          <SiteTheme />
+          <DraftModeScript appOrigin={process.env.MAKESWIFT_APP_ORIGIN} />
+        </head>
+        <body>
+          <Notifications />
+          <NextIntlClientProvider locale={locale} messages={messages}>
+            <NuqsAdapter>
+              <Providers>{children}</Providers>
+            </NuqsAdapter>
+          </NextIntlClientProvider>
+          <VercelComponents />
+        </body>
+      </html>
+    </MakeswiftProvider>
```

Comment:
Toggle the "Hide whitespace" setting to view the semantic diffs here:

<img width="711" alt="hide whitespace" src="https://github.com/user-attachments/assets/26daa5c0-a26a-4fd9-bac5-5a6d730c3750" />